hypothesis-python/src/hypothesis/extra/ghostwriter.py

```diff
@@ -952,6 +1116,9 @@ def magic(
             except (TypeError, ValueError):
                 pass
 
+    if annotations is None:
+        annotations = _are_annotations_used(*functions)
+
```

Comment:
Since we do this identically in every case, let's move it into `_make_test_body`.